src/main/java/uk/gov/crowncommercial/dts/scale/cat/service/JaggaerService.java

```diff
@@ -57,11 +57,11 @@ public CreateUpdateProjectResponse createUpdateProject(
    * @param rfx
    * @return
    */
-  public CreateUpdateRfxResponse createUpdateRfx(final Rfx rfx) {
+  public CreateUpdateRfxResponse createUpdateRfx(final Rfx rfx, final OperationCode operationCode) {
 
     var createRfxResponse =
         ofNullable(jaggaerWebClient.post().uri(jaggaerAPIConfig.getCreateRfx().get(ENDPOINT))
-            .bodyValue(new CreateUpdateRfx(OperationCode.CREATEUPDATE, rfx)).retrieve()
+            .bodyValue(new CreateUpdateRfx(operationCode, rfx)).retrieve()
```

Comment:
for consistency, we should use a builder pattern instead of a constructor

Reply:
Yep agree in principle - we've tended to migrate as needed on a case by case basis when we get errors (as originally the builder pattern wasn't used much, but has been expanding as we've been expanding the Jaggaer model)